cpp/dolfinx/mesh/graphbuild.cpp

```diff
@@ -266,10 +268,12 @@ compute_nonlocal_dual_graph(
   std::int64_t local_min = std::numeric_limits<std::int64_t>::max();
   std::int64_t local_max = 0;
 
-  for (std::size_t i = 0; i < facet_cell_map.shape[0]; ++i)
+  if (facet_cell_map.shape()[0] > 0)
   {
-    local_min = std::min(local_min, facet_cell_map(i, 0));
-    local_max = std::max(local_max, facet_cell_map(i, 0));
+    xt::xtensor<std::array<std::int64_t, 2>, 0> p
```

Comment:
This looks odd with `std::array` mixed in. Have you checked that it does what is expected?

Reply:
`xt::xtensor_fixed<std::int64_t, xt::xshape<2>> p` ?

Reply:
@IgorBaratta - Does that need another header? I can't get it to compile.
I agree the declaration looks odd, but it does return a scalar (0) dimension xtensor containing a std::array correctly.
I would be very happy with something else, but it is hard to determine the correct return type.

Reply:
I think it needs `xtensor/xfixed.hpp`, but std::array is simpler